tests/test-backend-ops.cpp

```diff
@@ -684,7 +684,11 @@ struct test_case {
         int total_runs = 0;
         do {
             int64_t start_time = ggml_time_us();
-            ggml_backend_graph_compute(backend, gf);
+            ggml_status status = ggml_backend_graph_compute(backend, gf);
+            if (status != GGML_STATUS_SUCCESS) {
+                fprintf(stderr, "%s: ggml_backend_graph_compute failed. status=%d \n", __func__, status);
+                return false;
+            }
```

Comment:
There are 9 calls to `ggml_backend_graph_compute` in this file alone, why only check this one? I don't expect everything to be done at the same time, but at least some consistency.

Reply:
I just did nt know if you wanted me to check all graphcompute() but I can do it if you d like me to.
On it...